diff --git a/.github/workflows/ci.yml b/.github/workflows/ci.yml
--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -1 +1,2 @@
 name: CI
+- uses: ./.github/actions/setup
diff --git a/.github/actions/setup/dist/index.js b/.github/actions/setup/dist/index.js
--- a/.github/actions/setup/dist/index.js
+++ b/.github/actions/setup/dist/index.js
@@ -1 +1,2 @@
 module.exports = {}
+module.exports.setup = true
